pages/Javascript-API.md

```diff
@@ -99,8 +90,8 @@ scene.screenshot().then(function(screenshot) { window.open(screenshot.url); });
 #### `setActiveCamera(camera)`
 Sets the active camera to the camera specified by name, as named in the scene file.
 
-#### `setDataSource(name, source)`
-Loads a new `[data source](sources.md)` or replaces an existing one, and rebuilds the scene.
+#### `setDataSource(_string_ name, _object_ config)`
+Loads a new `source` object (see `[sources](sources.md)`), returning a Promise which fulfills when the `source` is loaded.
```

Comment:
Note from earlier, it seems that the docs generator may not be working for links within backticks.